examples/rn-mbnb/test/backend/sync/config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "development_mode_enabled": false
```

Comment:
something funky might have happened with the CLI because sync is actually also not enabled in this config

Reply:
thanks, i'll fix this

Reply:
should work now

Reply:
Deployment worked all well now! But now I am getting 
```
failed to configure namespaces for sync: failed to configure namespace sample_mflix.PrivateContent for sync: error ensuring namespace exists: (AtlasError) you are over your space quota, using 514 MB of 512 MB
```
It might be because I am on the free tier? not sure if the intention is to make it fit in the free tier.

Reply:
I dropped some collections and I think that freed up some space, I still don't really see content in search but sync calls seem to go through

Reply:
Could drop the non-airbnb collections to fix this.  Wish there was a way to constrain which sample data sets one would like to load into the cluster.

Reply:
doh...i see the issue. the backend is still not really correct.  I'll have to do another push.